tcmur_work.c

```diff
@@ -41,16 +41,11 @@ struct tcmur_work *tcmur_create_work(void)
 
 static void __tcmur_flush_work(struct tcmur_work *work)
 {
-	char pname[TCMU_THREAD_NAME_LEN];
-
-	if (pthread_getname_np(pthread_self(), pname, TCMU_THREAD_NAME_LEN))
-		return;
-
 	/*
 	 * The event work thread may need to do a handler reopen
 	 * call and try to flush itself. Just ignore.
 	 */
-	if (!strcmp(pname, "ework-thread"))
+	if (__tcmu_is_ework_thread)
```

Comment:
This seems won't work.
This function will also be call in none `ework-thread` thread.

Reply:
Could you please elaborate why? I'm not C-coder.

Reply:
Oh. I should negate this. My bad, sorry.

Reply:
Yeah, you're right.